go.mod

```diff
@@ -31,8 +31,10 @@ require (
 	github.com/spf13/viper v1.4.0
 	github.com/stretchr/testify v1.5.1
 	go.uber.org/zap v1.14.1
+	golang.org/x/lint v0.0.0-20200302205851-738671d3881b // indirect
```

Comment:
We do not need usually add the indirect. could you please remove and run `go mod tidy` to check?

Reply:
I ran `go mod tidy`, still they persist.

Reply:
The `go mod tidy` sometimes do not remove. You can remove manually and then run `go mod tidy` to ensure. 
You can also run `go mod why golang.org/x/lint v0.0.0-20200302205851-738671d3881b`. If its output **contains** (main module does not need package v0.0.0-20200302205851-738671d3881b) it can be removed safely.